app/helpers/catalog_helper.rb

```diff
@@ -12,7 +12,7 @@ module CatalogHelper
   # @return [Boolean]
   # rubocop:disable Style/PredicateName
   def has_thumbnail?(document)
-    return super unless document.reference?
+    return super unless document.reference? || document.canvas?
```

Comment:
I feel like there is a pattern emerging here but I don't know exactly what to call it yet...